app/helpers/spree/admin/slides_helper.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Admin
+    module SlidesHelper
+      def get_column_header_by_type(type)
```

Comment:
let's add here guard clause and handle situation when `type` isn't `image/product` in other case this method will return `String` || `nil`.

Reply:
It now returns '----' in other cases